dev/Updates/powermod-upoly

```diff
@@ -0,0 +1,28 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Format 'yyyy/mm/dd'
+2015/4/1
+
+!! Changed by
+SL
+! Reported by
+Max Hown
```

Comment:
I've added Travis CI file to the `stable-4.7` branch so we should be able to test this PR now. Could you please amend it (for example, fix the typo in the line above) and push the change to update the PR and trigger a new Travis build?